channels/encomsp/server/encomsp_main.c

```diff
@@ -174,7 +197,7 @@ static void* encomsp_server_thread(void* arg)
 
 	while (1)
 	{
-		status = WaitForMultipleObjects(nCount, events, FALSE, INFINITE);
+		WaitForMultipleObjects(nCount, events, FALSE, INFINITE);
```

Comment:
I think the return value should be checked here. In case of memory shortage WaitForMultipleObjects might return an error.